docs/rules/order.md

```diff
@@ -100,11 +105,11 @@ To be able to group by paths mostly needed with aliases pathGroups can be define
 
 Properties of the objects
 
-| property       | required | type   | description   |
-|----------------|:--------:|--------|---------------|
-| pattern        |     x    | string | minimatch pattern for the paths to be in this group (will not be used for builtins or externals) |
-| patternOptions |          | object | options for minimatch, default: { nocomment: true } |
-| group          |     x    | string | one of the allowed groups, the pathGroup will be positioned relative to this group |
+| property       | required | type   | description                                                                                                                                              |
+| -------------- | :------: | ------ | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| pattern        |    x     | string | minimatch pattern for the paths to be in this group (will not be used for builtins or externals)                                                         |
+| patternOptions |          | object | options for minimatch, default: { nocomment: true }                                                                                                      |
+| group          |    x     | string | one of the allowed groups, the pathGroup will be positioned relative to this group                                                                       |
```

Comment:
i'm not sure why this table was changed at all (perhaps prettier running on a project that doesn't use it?) but please revert it.

Reply:
Will do